orttraining/orttraining/core/graph/optimizer_builder.h

```diff
@@ -150,7 +150,7 @@ class OptimizerBuilder {
   }
 
  private:
-  std::string name_;
+  OpDef op_def_;
```

Comment:
can `op_def_` and `attr_names_` be const?

Reply:
done